omni/omni.py

```
@@ -0,0 +1,865 @@
+            global_image_count=0,
+            global_moderated_image_count=0,
+            global_timeout_count=0,
+            global_total_timeout_duration=0
+        )
+
+    async def initialize(self):
+        """Initialize the aiohttp session."""
+        try:
+            self.session = aiohttp.ClientSession()
+        except Exception as e:
+            raise RuntimeError(f"Failed to initialize Omni cog: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)